packages/lit-ssr/src/lib/util/iterator-readable.ts

```diff
@@ -0,0 +1,39 @@
+/**
```

Comment:
How is this different from https://nodejs.org/api/stream.html#stream_stream_readable_from_iterable_options?

Reply:
It might not be, that just might not have existed when I wrote this. This was originally async too...

Reply:
Good call. I'm guessing it didn't exist when Justin started on this? Dunno, but swapped that in and got rid of this file.